packages/editor/src/plugins/components/ToolBar.tsx

```diff
@@ -0,0 +1,229 @@
+import * as React from "react";
+// @ts-ignore
+import styled, {StyledComponentClass} from "styled-components";
+import {
+  FormatListBulleted,
+  FormatListNumbered,
+  FormatQuote,
+  Title,
+  FormatBold,
+  FormatItalic,
+  InsertLink,
+  FormatStrikethrough,
+  InsertPhoto,
+} from "material-ui-icons";
+import { ImageModal, ImageModalState } from "./ImageModal";
+
+import { constants, Plugin } from "../index";
+
+export interface ToolBarProps {
+  readOnly: boolean;
+  value: any;
+  editor: any;
+  onChange: any;
+  plugins: Plugin[];
+}
+
+export interface MarkObject {
+  type: string;
+  data: any;
+}
+
+export interface ToolBarState {
+  openImageModal: boolean;
+}
+
+const ToolBarWrapper = styled.div`
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  right: 10vw;
+  top: 20px;
+  z-index: 10;
+  svg, button {
+    cursor: pointer;
+    padding: 10px;
+  }
+  & path {
+    stroke: #2B56FF;
+    fill: #2B56FF;
+  }
+  & .active path {
+    stroke: #30E8BD;
+    fill: #30E8BD;
+  }
+`;
+
+const Button = styled.button`
+  background: transparent;
+  font-family: "Libre Franklin", sans-serif;
+  font-weight: 700;
+  color: #2B56FF;
```

Comment:
so I think the answer to this is "we'll sort it out later", but there are a lot of hard coded colors in the styling. In SASS i would import a vars file, but not sure how it works in styled components. Is this something you plan to address in the future?

Reply:
oh I had a note to myself to move them all to a constants file, but i forgot anyways

Reply:
in the future we should probably import them from another package, so they can stay consistent across products